schemas/ethereum.schema.graphql

```diff
@@ -54,6 +54,15 @@ type MasterData @entity {
   total_bonus_out: BigDecimal!
   init_unlocked_percent: BigDecimal!
   global_start_time: Int!
+  gtoken_utilization:BigDecimal!
+}
+
+type TrancheBalance @entity {
+  id: ID! # hash-logIndex
+  timestamp: Int!
+  junior_tranche_balance: BigDecimal!
+  senior_tranche_balance:BigDecimal!
+  utilization: BigDecimal!
 }
```

Comment:
This might be removed if we don't need all transactions, but only the latest values. If so, they can be part of the Master Data.